images_config/README_images.md

```diff
@@ -0,0 +1,395 @@
+```
+openssl passwd -1 yourpassword 
+```
+in Terminal to generate the new password. Place the generated new password in the place of Password and Confirm Password.
+
+![alt text](doc/kickstart-7.png "Logo Title Text 7")
+Keep it disabled. Ubuntu doesn't support firewall settings. 
+
+
+![alt text](doc/kickstart-8.png "Logo Title Text 8")
+Do not configure the X Window System here. Ubuntu automaticlly solves this one anyway.
+
+
+![alt text](doc/kickstart-9.png "Logo Title Text 9")
+Ubuntu doesn't support Kickstart package selection. We will add them manually %packages section in ks.cfg file.
+
+![alt text](doc/kickstart-10.png "Logo Title Text 10")
+We have some specific per-installation script, we will add manually in %pre section in ks.cfg file.
+
+![alt text](doc/kickstart-11.png "Logo Title Text 11")
+We have some specific post-installation script, we will add manually in %post section in ks.cfg file.
+
+
+Write anything that you need to do after kickstart installation. It executes the script in chroot environment, so you don't need to use sudo.
+
+When you are finished with the configuration, press File > Save File in the top menu. Keep default name as ks.cfg and save it to your Desktop.
+
+In ks.cfg search for #Disk partitioning information where we are going make small changes in the partition according to our requirement. Remove the lines of partition and paste below mentioned lines.
+```
+#Disk partitioning information 
+##################################################### 
+clearpart --all --initlabel 
+part /boot --fstype ext4 --size 200 --asprimary 
+part swap --size 1024 
+part pv.01 --size 1 --grow 
+volgroup rootvg pv.01 
+logvol / --fstype ext4 --vgname=rootvg --size=1 --grow --name=rootvol 
+# needed to answer the 'do you want to write changes to disk" 
+preseed partman-lvm/confirm_nooverwrite boolean true 
+
+# needed to answer the question about not having a separate /boot 
+preseed partman-auto-lvm/no_boot boolean true 
+#####################################################
+```
+In automatic installation of Care-O-bot we are making two ISO images, one for Slave and one for Master.
+Master Machine: cob4-X-b1
+Slave Machines: cob4-X-t1, cob4-X-t2, cob4-X-t3, cob4-X-s1, cob4-X-h1
+
+The ks.cfg which is saved on Desktop make copy of same file, name one with ks-robot-slave.cfg and other with ks-robot-master.cfg. Now we have two configuration file by which we can extract two image files one for Master and one for Slave.
+### 3. Adding packages, pre-installation, post-installation for Master and Slave configuration files. <a id="Adding packages, pre-installation, post-installation for Master and Slave configuration files."/> 
+### a. Packages: <a id="Packages"/> 
+Use the %packages command to begin a Kickstart section which describes the software packages to be installed. 
+```
+Packages for ks-robot-master.cfg:
```

Comment:
please don't copy the code, just use a link to the lines:
https://github.com/ipa320/setup_cob4/blob/master/images_config/ks-robot-master.cfg#L53-L68